test/PerfTests/JavaScript/ManyInstances/index.js

```diff
@@ -0,0 +1,14 @@
+﻿const df = require("durable-functions");
+
+module.exports = async function (context, req) {
+    const client = df.getClient(context);
+    const numInstances = req.body;
+
+    for(var i = 0; i < numInstances; i++)
+    {
+        var instanceId = await client.startNew("ManyInstancesOrchestrator");
```

Comment:
In C# we start multiple instances in parallel, whereas here you're starting them sequentially. It's not a blocker, but beware that you'll more easily run into HTTP timeouts with this approach when starting large number of instances. More info here: https://dev.to/cgillum/scheduling-tons-of-orchestrator-functions-concurrently-in-c-1ih7